docs/sources/panels/transformations/types-options.md

```diff
@@ -402,6 +402,8 @@ Conditions that are invalid or incompletely configured are ignored.
 
 ## Rename by regex
 
+> **Note:** This transformation is only available in Grafana 7.4+.
```

Comment:
Small nit: please remove "only".

Reply:
Great feedback!

But I have already merged the PR and that line is repeated on multiple places in the docs to I guess you should open an issue regarding this so it can be updated on all places.

Reply:
+1